### PR TITLE
Add script to write notes to couchdb and update other scripts that write running notes

### PR DESCRIPTION
**Pull request by aanil**
Work in progress.

### CODE REVIEW

File: scripts/make_running_note_from_pooling.py [hunk 1]
```diff
@@ -42,20 +44,17 @@ def main(lims, args):
...
         if len(datamap[pid]) > 1:
             rnt="{0} samples planned for {1}".format(len(datamap[pid]), wsname)
         else:
             rnt="{0} sample planned for {1}".format(len(datamap[pid]), wsname)
 
-        running_notes[now]={"note": rnt, "user" : username, "email":user_email, "category":"Workset"}
-
-        pj.udf['Running Notes']=json.dumps(running_notes)
-        pj.put()
+        running_note = {"note": rnt, "user" : username, "email":user_email, "category":"Workset"}
```
Comment (alneberg) [inline scripts/make_running_note_from_pooling.py:52]: I wouldn't mind a even more of a whitespace cleanup here. Removing whitespace after `"user"` and adding a space after `"category:"`

File: scripts/write_notes_to_couchdb.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+DESC="""Common script called by other EPP scripts to write notes to couchdb
```
Comment (alneberg) [inline scripts/write_notes_to_couchdb.py:2]: Module rather than "common script"?

File: scripts/write_notes_to_couchdb.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import os
+from email.mime.text import MIMEText
+
+
+def write_note_to_couch(pid, timestamp, note, lims):
+    configf = '~/.statusdb_cred.yaml'
+    with open(os.path.expanduser(configf)) as config_file:
+        config = yaml.safe_load(config_file)
+    if not config['statusdb']:
+        print('Statusdb credentials not found')
```
Comment (alneberg) [inline scripts/write_notes_to_couchdb.py:17]: Maybe this should be an email error as well?

File: scripts/make_running_note_from_pooling.py (hunk 1 above)
Comment (aanil) [reply scripts/make_running_note_from_pooling.py]: Done!

File: scripts/write_notes_to_couchdb.py (hunk 2 above)
Comment (aanil) [reply scripts/write_notes_to_couchdb.py]: Done!

File: scripts/write_notes_to_couchdb.py (hunk 3 above)
Comment (aanil) [reply scripts/write_notes_to_couchdb.py]: Done!

**Review by chuan-wang — APPROVED** [review]
👍

File: scripts/write_notes_to_couchdb.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        sys.exit(1)
+    url_string = 'http://{}:{}@{}:{}'.format(config['statusdb'].get('username'), config['statusdb'].get('password'),
+                                              config['statusdb'].get('url'), config['statusdb'].get('port'))
+    couch = couchdb.Server(url=url_string)
+    if not couch:
+        email_error('Connection failed from {} to {}'.format(lims, config['statusdb'].get('url')), 'genomics-bioinfo@scilifelab.se')
+        email_error('Running note save for {} failed on LIMS! Please contact {} to resolve the issue!'.format(pid, 'genomics-bioinfo@scilifelab.se'), note['email'])
+
+    proj_db = couch['projects']
+    doc_id = proj_db.view('project/project_id')[pid].rows[0].value
```
Comment (alneberg) [inline scripts/write_notes_to_couchdb.py:28]: Wonder if this works if the project is not found (as tested for below). I guess `rows[0]` would fail?

**Pull request merged**